params/config.go

```diff
@@ -820,6 +820,11 @@ func (c *ChainConfig) ElasticityMultiplier() uint64 {
 	return DefaultElasticityMultiplier
 }
 
+// Is4844Enabled checks whether blob transactions are supported.
+func (c *ChainConfig) Is4844Enabled() bool {
+	return false
+}
```

Comment:
Can we directly remove it? We want to minimize the changes in geth repo so it is easier for us to incorporate changes from geth main repo.